docs/insomnia/template-tags.md

```diff
@@ -5,31 +5,37 @@ category: "Plugins"
 category-url: plugins
 ---
 
-Template tags are closely related to Environment Variables. They can be used in the same places and they behave in a similar way. The main difference is that template tags are more like operations, not variables. Tags can do things like transform strings, random numbers, UUIDs, timestamps, and so on.
+Template tags are closely related to [Environment Variables](/insomnia/environment-variables/). The main difference between Template Tags and Environment Variables is that Template Tags act more like operations, and not variables.
 
-This article will cover the basics of using template tags, tells you how you can create your own, and goes over two of the most powerful tags that Insomnia includes out of the box.
+Tags can do things like transform strings, generate random numbers, handle UUIDs, and create timestamps.
 
-## Using Template Tags
+## Use Template Tags
 
-To insert a template tag, press CTRL+Space wherever Environment Variables can be used. Template tags will appear below environment variables in the autocomplete list and are marked with an ƒ symbol.
-
-{:.alert .alert-primary}
-**Note**: The autocomplete will also be shown based on what you type.
+To insert a template tag, press CTRL+Space wherever [Environment Variables](/insomnia/environment-variables/) can be used. Template tags will appear below Environment Variables in the autocomplete list, and are marked with an `ƒ` symbol.
```

Comment:
```suggestion
To insert a template tag, press CTRL+Space wherever [Environment Variables](/insomnia/environment-variables/) can be used (most text fields). Template tags will appear below Environment Variables in the autocomplete list, and are marked with an `ƒ` symbol.
```

I don't think we mention where environment variables can be used so this feels like deflection? The basic note is that most (but definitely not all) text fields allow for environment variables and template tags.

Reply:
See note above. I don't have enough information to expand on the current docs.